types/material-ui/index.d.ts

```diff
@@ -986,6 +986,7 @@ declare namespace __MaterialUI {
             inputStyle?: React.CSSProperties;
             onBlur?: React.FocusEventHandler<{}>;
             onKeyDown?: React.KeyboardEventHandler<{}>;
+            openToYearSelection?: boolean;
```

Comment:
[This commit](https://github.com/mui-org/material-ui-pickers/commit/0cc4fedac0471af33381d0c33686ee39a2eb8939#diff-4cdce62a1de4bfce2f4f538dd2691344) deprecates this property.

Reply:
Actually I was looking at the new library, which comes with TS definitions, instead of old one. This looks fine (found it in https://unpkg.com/browse/material-ui@0.20.2/DatePicker/DatePicker.js).